main/test/ca/uwaterloo/flix/language/feature/Test.Expression.Select.flix

```diff
@@ -0,0 +1,85 @@
+/////////////////////////////////////////////////////////////////////////////
+// Select expressions (channel select)                                     //
+/////////////////////////////////////////////////////////////////////////////
+
+def main(): Int = 42
+
+// Test integers
+def fi01(): Int = select {
```

Comment:
These should simply be named `testSelect01` or `testSelectChannel01` and so on.

Reply:
Also, the test cases need to be positive (i.e. complete) so you cannot use `ch` if it is not defined.